runtime/zeitgeist/src/parameters.rs

```diff
@@ -288,6 +288,35 @@ parameter_types! {
     /// Pallet identifier, mainly used for named balance reserves. DO NOT CHANGE.
     pub const TreasuryPalletId: PalletId = PalletId(*b"zge/tsry");
 
+    // Bounties
+    /// The amount held on deposit for placing a bounty proposal.
+    pub const BountyDepositBase : Balance = 10 * BASE;
+    /// The delay period for which a bounty beneficiary need to wait before claim the payout.
+    pub const BountyDepositPayoutDelay : BlockNumber = 3 * BLOCKS_PER_DAY;
+
+    /// Bounty duration in blocks.
+    pub const BountyUpdatePeriod : BlockNumber = 14 * BLOCKS_PER_DAY;
+
+    /// The curator deposit is calculated as a percentage of the curator fee.
+    ///
+    /// This deposit has optional upper and lower bounds with `CuratorDepositMax` and
+    /// `CuratorDepositMin`.
+    pub const CuratorDepositMultiplier : Permill = Permill::from_percent(50);
+
+    /// Maximum amount of funds that should be placed in a deposit for making a proposal.
+    pub const CuratorDepositMax : Balance = 200 * BASE;
+    /// Minimum amount of funds that should be placed in a deposit for making a proposal.
+    pub const CuratorDepositMin : Balance = 10 * BASE;
+    /// Minimum value for a bounty.
+    pub const BountyValueMinimum: Balance = BASE;
+
+    /// The amount held on deposit per byte within the tip report reason or bounty description.
+    pub DataDepositPerByte : Balance = BASE;
+    /// Maximum acceptable reason length.
+    ///
+    /// Benchmarks depend on this value, be sure to update weights file when changing this value
+    pub MaximumReasonLength : u32 = 256;
```

Comment:
Unsure about this one. This is the maximum length of the description of a bounty. We can use 256 or 512, if we want to establish a system where the descriptions are stored in IPFS. I think long term this is the way to go, short term I think we need resources in other places than implementing this feature (submit bounty descriptions in IPFS and decode in frontend).
For now I propose we allow 8kB. This is fine because the proposer reserves `DataDepositPerByte` per byte. Later we switch to IPFS CID length.
```suggestion
    pub MaximumReasonLength : u32 = 8192;
```